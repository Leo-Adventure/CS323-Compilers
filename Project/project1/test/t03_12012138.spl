int t03_12012138() 
{
    float right_float = 1.234;
    int wrong_int_1 = 0x023456;
    int wrong_int_2 = 023456;
    int right_int = 23456;
    char right_char = 'c';
    char wrong_char = 'wrong_char';
    char wrong_char = '\xuu';
}
