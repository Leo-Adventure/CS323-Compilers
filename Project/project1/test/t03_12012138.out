Error type A at Line 8: unknown lexeme 'wrong_char'
Error type A at Line 9: unknown lexeme '\xuu'
